docs/Installation/topics/adding_CIFS_storage.rst

```diff
@@ -0,0 +1,118 @@
+Cosmos CIFS Use-Case
+====================
+
+By design, storage backend integration into Zenko and Cloudserver are
+only limited by the type of volumes supported by Kubernetes.
+Fortunately, Kubernetes supports “FlexVolume” drivers that allow for
+custom volume driver integration into the Kubernetes ecosystem. This
+guide uses the Azure CIFS/SMB FlexVolume driver for Kubernetes.
+https://github.com/Azure/kubernetes-volume-drivers/tree/master/flexvolume/smb
+
+CIFS/SMB Install on MetalK8s
+============================
+
+Install dependencies and FlexVolume driver on every VM MetalK8s VM
+------------------------------------------------------------------
+
+.. code:: bash
+
+   yum install epel-release -y
+   yum install wget jq cifs-utils -y
+
+   export PLUGINS=/var/lib/kubelet/volume-plugins/microsoft.com~smb
+   mkdir -p $PLUGINS
+   wget -O $PLUGINS/smb https://raw.githubusercontent.com/Azure/kubernetes-volume-drivers/master/flexvolume/smb/deployment/smb-flexvol-installer/smb
+   chmod a+x $PLUGINS/smb
+
+Usage with Cosmos Integration
+=============================
+
+Installing the chart with a Zenko instance
+------------------------------------------
+
+In order to configure Cosmos with a Zenko instance, you will need to
+perform the following steps in `Orbit <https://admin.zenko.io>`__:
+
+1. Create a storage location of type “NFS Mount”.
+-------------------------------------------------
+
+In the CIFS/SMB use-case the NFS protocol and NFS version are ignored
+and all configuration is done via config file below.
+
+2. Create a bucket within this location constraint.
+---------------------------------------------------
+
+3. Create a user for Cosmos.
+----------------------------
+
+4. Use information from the previous steps to configure instance-specific Cosmos values. Export the following variables with appropriate values entered:
+--------------------------------------------------------------------------------------------------------------------------------------------------------
+
+.. code:: bash
+
+   # Values from Orbit
+   export ACCESS_KEY=<your-cosmos-user-access-key>
+   export SECRET_KEY=<your-cosmos-user-secret-key>
+   export SMB_BUCKET=<your-cosmos-bucket-name>
+   export SMB_LOCATION=<your-nfs-mount-location-name>
+
+   # Values of your NFS mount point
+   export SMB_HOST=<your-smb-host>
+   export SMB_PATH=<your-smb-path>
+
+   # Cloudserver endpoint (assuming it's running on the same namespace)
+   export CLOUDSERVER_ENDPOINT="http://$(kubectl get svc -l app=cloudserver -o jsonpath='{.items[*].metadata.name}')"
+
+5. Create a secret to store the CIFS/SMB account name and password
+------------------------------------------------------------------
+
+.. code:: bash
+
+   kubectl create secret generic smbcreds --from-literal username=<USERNAME> --from-literal password="<PASSWORD>" --type="microsoft.com/smb"
```

Comment:
`<USERNAME>` doesn't need to be quoted to be consistent with `"<PASSWORD>"`

Reply:
I'll make both quoted. I had issues when testing with special characters that weren't quoted.